source/MaterialXTest/ShaderValid.cpp

```diff
@@ -1112,7 +1112,7 @@ bool getTestOptions(const std::string& optionFile, ShaderValidTestOptions& optio
 
 void printRunLog(const ShaderValidProfileTimes &profileTimes, const ShaderValidTestOptions& options,
     std::set<std::string>& usedImpls, std::ostream& profilingLog, mx::DocumentPtr dependLib,
-    mx::OslShaderGeneratorPtr oslShaderGenerator, mx::GlslShaderGeneratorPtr glslShaderGenerator,
+    mx::ArnoldShaderGeneratorPtr oslShaderGenerator, mx::GlslShaderGeneratorPtr glslShaderGenerator,
```

Comment:
This is incorrect: You should not revert this back to Arnold. There should be no arnold dependency in this test.

Reply:
Ya, it looks like somehow I merged things incorrectly, this whole file seems wrong.